environment.yml

```diff
@@ -0,0 +1,11 @@
+name: test-environment
+channels:
+- mosdef
```

Comment:
You're right, we don't need any of it. I was thinking since ```msibi``` is now hosted on ```mosdef-hub``` it needed to be there but we aren't installing it from conda so it's not needed.